<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastroExame</class>
 <widget class="QWidget" name="CadastroExame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastrar Exame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitulo">
     <property name="text">
      <string>Cadastrar Exame</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelData">
     <property name="text">
      <string>Data do Exame:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="inputData">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy HH:mm</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDescricao">
     <property name="text">
      <string>Descrição:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="inputDescricao">
     <property name="placeholderText">
      <string>Digite a descrição do exame</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="botaoSelecionarCSV">
     <property name="text">
      <string>Selecionar CSV</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelCSVSelecionado">
     <property name="text">
      <string>CSV selecionado: Nenhum</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="botaoCadastrar">
     <property name="text">
      <string>Cadastrar Exame</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="botaoVoltar">
     <property name="text">
      <string>Voltar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
</ui>
